### ISSUE TITLE
Reaper setting "Auto float newly added FX window" messes with MFXlist...

### ISSUE THREAD

**Issue by martinfabian** [martinfabian/MFXlist#33]
If the user has the option to "Auto-float newly added FX window" turned on, then when the user adds a new FX via the right-click menu, it seems that the code to toggle the window sees the window open (because Reaper just opened it) and so MFXlist closes the window. This is not as expected.
So, when an FX is added via the right-click menu, MFXlist should check if the floating window is open, and in that case not do anything, else if the window is not open, MFXlist should open it.
This is probably manageable by an extra parameter to the handleToggleWindow function; if the parameter is true then the window should not be closed if it is open.

**Comment by martinfabian** [comment]
A better solution is not using `handleToggleWindow `in `handleMenu`, but instead simply show the floating FX (and adding it to the `openwin_list`). that is:

```
-- handleToggleWindow(track, index, 2, true) -- true => if window already opened, then don't close it
rpr.TrackFX_Show(track, index, 3) -- 3 == show floating window
MFXlist.openwin_list:insert({track, index})
```

With the auto-float setting on, Reaper auto-floats the FX, and asking to open again the floating FX window seems to work as expected. With teh auto-float setting off, Reaper does not auto-float the FX but instead opens the floating window on the Show-call.

**Comment by martinfabian** [comment]
Solved in f170487
